### DISCUSSION TITLE
Uploading Data to Collection

### DISCUSSION THREAD

**Discussion by Kantoborgy** [Symbiota/community-board#399] [Announcements]
Hi all,
I have the following error when trying to upload new data in a new collection (full text file import). Before the last code update(2023-09-08) , data uploading was working perfectly . My server, Ubuntu 22, MariaDB 15, PHP 8.

Thank you in advance
All the best

Data Upload Module
JARDIN BOTANICO DE LOJA
Last Upload Date: not recorded
Upload Status:
Initiating import from: occurrence-jb.csv
Clearing staging tables
Beginning to load records...
Count: 1000
Data cleaning:
Cleaning event dates...

[Mon Oct 02 08:57:01.155692 2023] [php:error] [pid 108960] [client 72.90.175.164:49705] PHP Fatal error: Uncaught mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'IN(31))' at line 1 in /var/www/html/l/classes/SpecUploadBase.php:635\nStack trace:\n#0 /var/www/html/l/classes/SpecUploadBase.php(635): mysqli->query()\n#1 /var/www/html/l/classes/SpecUploadBase.php(594): SpecUploadBase->recordCleaningStage1()\n#2 /var/www/html/l/classes/SpecUploadFile.php(144): SpecUploadBase->cleanUpload()\n#3 /var/www/html/l/collections/admin/specuploadprocessor.php(132): SpecUploadFile->uploadData()\n#4 {main}\n thrown in /var/www/html/l/classes/SpecUploadBase.php on line 635, referer: http://www.bioloja.org/l/collections/admin/specuploadmap.php

**Comment by themerekat** [comment]
Hi @Kantoborgy ! This seems like a question specific to your installation, so I'm moving it here to Discussions; however, I think it would best be served by emailing our support team at help@symbiota.org

**Reply by Kantoborgy** [reply]
Thank you. I already sent an email
